conf/supervisord-dev.conf

```diff
@@ -1,6 +1,8 @@
 [supervisord]
-nodaemon = true
-silent = true
+nodaemon=true
+environment=PYTHONUNBUFFERED="1"
+# logfile=/dev/null
+# logfile_maxbytes=0
```

Comment:
?

Reply:
I was trying to get the files to be as similar as possible, rather than previously where every line was different and swapping between them made it very difficult to spot differences